src/main/java/bibliotecame/back/Book/BookController.java

```diff
@@ -81,8 +81,13 @@ public BookController(BookService bookService, TagService tagService, CopyServic
             if(copies.size()>=100) return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
 
             for(CopyModel copy : copies){
-                if(copyService.exists(copy.getId())) return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
-                savedCopies.add(copyService.saveCopy(copy));
+                if(copyService.exists(copy.getId())){
+                    if ((copy.getBooked() && !copy.getActive())) return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
```

Comment:
What is checking this line? If the copy is booked and is not active? (Is that possible?)